eng/cibuild.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Curious the value of this versus utilizing the existing build.sh script at the repo root?  The only difference I see is the addition of `--test --pack --publish --ci` when invoking the eng/common build script.

Reply:
I did this to keep consistency with the existing divide between the build.sh/cibuild.sh scripts.

Reply:
If that's the reason, would it make sense to have this call `. "$scriptroot/common/build.sh" $@` instead of `. "$scriptroot/common/build.sh" --restore --build --test --pack --publish --ci $@`?

Reply:
I meant to say `. "$scriptroot/common/buildci.sh" $@`

Reply:
Ah, good point.  Fixed.